docs/blockchain/contracts/nft-contract/nft-actions/setprchsreq.a.mainnet.md

```diff
@@ -10,11 +10,9 @@ This action is used to set purchase requirements for a token factory.
 
 Tokens purchased will be issued to the receiver account using [issue.b](./issue.b.md) action. Factory manager pays for minting the token
 
-<Mainnet>
 ::: warning
 This action is currently disabled.
```

Comment:
On mainnet this action currently is not disabled

Reply:
From I I cheked, it is disabled.
```
$ cleos -u http://ultra.api.eosnation.io get table ultra.cntmgr eosio.nft.ft disabledact
{
  "rows": [{
      "action": "setprchsreq.a"
    },{
      "action": "settrdwin.a"
    },{
      "action": "settrnfwin.a"
    }
  ],
  "more": false,
  "next_key": ""
}
```